tests/integration/test_dss.py

```diff
@@ -8,23 +9,36 @@
 from lightkube.resources.core_v1 import Namespace, PersistentVolumeClaim, Service
 
 from dss.config import DSS_CLI_MANAGER_LABELS, DSS_NAMESPACE, FIELD_MANAGER
+from dss.utils import KUBECONFIG_ENV_VAR
 
 # TODO: is there a better way to initialize this?  Maybe an optional argument to the test?
-KUBECONFIG = "~/.kube/config"
 NOTEBOOK_RESOURCES_FILE = "./tests/integration/notebook-resources.yaml"
 NOTEBOOK_NAME = "test-nb"
 NOTEBOOK_IMAGE = "kubeflownotebookswg/jupyter-scipy:v1.8.0"
 
 
-def test_status_before_initialize(cleanup_after_initialize) -> None:
+@pytest.fixture()
+def set_dss_kubeconfig_environment_variable(monkeypatch):
```

Comment:
I mentioned this also above we are never testing iinmy opinion mostly common usecase for dss initialize

```
dss initialize --kubeconfig="$(microk8s config)"
```

I think we should have also test where we are actually sending the kubeconfig as text.

Reply:
Yeah I see what you mean.  At minimum, we do need to test this method of calling `dss initialize` somewhere as we currently dont at all

Reply:
I really like it :)